src/ui/services/Background.ts

```diff
@@ -558,10 +558,11 @@ export enum WalletTypes {
   KeystoreWx = 'keystore_wx',
 }
 
-export interface AssetDetail extends IAssetInfo {
+export interface AssetDetail extends Asset {
```

Comment:
This interface defines the structure of plain objects, not instances of classes, which is why `IAssetInfo` was previously used.

Reply:
`AssetDetails`  determines only fields of plain object. 
That means in functions that returns this type is not instances of `Asset` class (cuz they don't have methods implementation)